include/bluetooth/gatt.h

```diff
@@ -338,6 +338,10 @@ enum {
 /** @typedef bt_gatt_attr_func_t
  *  @brief Attribute iterator callback.
  *
+ *  Note: Attributes passed to this callback are temporary when it happens to
```

Comment:
This may be due to a lack of knowledge of gatt.c, but I do not fully understand this comment.
I think "Attributes passed to this callback are temporary if the attribute happens to be in the static portion of the database, so the attribute can not be used as the original location" is clearer.  (Gets rid of the "it" to refer to the "attributes" in plural.)
But "can not be used the original location" is still not clear to me.  (I understand that it is temporary, and therefore should not be used.  but as the "original location"?)